### DISCUSSION TITLE
Is it possible to use the Hyper key (cmd + ctrl + shift + opt) as a modifier?

### DISCUSSION THREAD

**Discussion by ddanieltan** [philc/vimium#4571] [General]
If so, what is the syntax?

**Comment by philg-dev** [comment]
Judging from the Readme, I highly doubt that's possible. I think Vimium only supports key-combinations with a single modifier at a time.

https://github.com/philc/vimium?tab=readme-ov-file#custom-key-mappings

**Reply by ddanieltan** [reply]
Noted, thanks for responding.
